README.md

```diff
@@ -32,29 +31,11 @@ docker compose up build_deployment_package
 
 This will output the deployment zip file in the root directory.
 
-## Running the Lambda locally ##
-
-The configuration in this repository allows you run the Lambda locally for
-testing as long as you do not need explicit permissions for other AWS
-services. This can be done with the following command:
-
-```console
-docker compose up --detach run_lambda_locally
-```
-
-You can then invoke the Lambda using the following:
-
-```console
- curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}'
-```
-
-The `{}` in the command is the invocation event payload to send to the Lambda
-and would be the value given as the `event` argument to the handler.
-
-Once you are finished you can stop the detached container with the following command:
+To customize the name of the deployment file, you can override the
+`BUILD_FILE_NAME` environment variable.  For example:
 
 ```console
-docker compose down
+BUILD_FILE_NAME="publish_egress_ip_lambda.zip" docker compose up build_deployment_package
```

Comment:
I'm fine with mentioning both ways.  @mcdonnnj Can you write some verbiage about this in [skeleton-aws-lambda-python](https://github.com/cisagov/skeleton-aws-lambda-python) and then it can filter to all of the descendants?

Reply:
Good idea. I created https://github.com/cisagov/skeleton-aws-lambda-python/issues/26 to track this work.